### DISCUSSION TITLE
Looking for advice on autoreloading after deploys

### DISCUSSION THREAD

**Discussion by msurdi** [nuxt/nuxt#9780] [Questions (Nuxt 2)]
Hi!

What's the recommended way in the Nuxt ecosystem to reload the application when there is a new version deployed? Currently I'm having the issue of users with an old version of the application getting 404 from bundles when navigating between pages, because they're gone in the new release.

Thanks!

**Comment by dmitryburov** [comment]
It's individual ... perhaps
I'm use version check hash (every 10 min).
He always auto-generated before deploy.

After deploy, there are 2 variants:
1. reload at once
2. show message of update

I'm using second =)
![image](https://user-images.githubusercontent.com/11191484/133455425-45a2793a-f6b9-4aed-ab7b-4fb46fe38c8a.png)
